pages/[role]/[id].tsx

```diff
@@ -577,6 +579,21 @@ const UserProfile: React.FunctionComponent<gsspProps> = ({
       } else {
         setUser((await response.json()).user);
         refreshData();
+        let toastMessage;
+        if (user?.status === UserStatus.Inactive) {
+          toastMessage = "User account deactivated";
```

Comment:
i think these messages are switched if the status if Inactive it should be "User account activated" and then vise versa!

Reply:
oops completed!